release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -74,6 +74,8 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
```

Comment:
Please include a title for this release note.

Reply:
Updated as suggested.